pages/account/billing.vue

```diff
@@ -99,6 +99,7 @@
   import ButtonAsync from '@/components/ButtonAsync.vue';
   import StripeCard from '@/components/StripeCard.vue';
   export default {
+    middleware: 'auth-guard',
```

Comment:
Is there a reason we have the middleware on all of the sub-pages?  I've tested removing it, and the one on /account seems to pick up everything just fine.